<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>1184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="MainWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QGroupBox" name="DataAreaGroup">
      <property name="title">
       <string>Data Area</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_19">
       <item>
        <widget class="QGroupBox" name="ZAxesForceGroup">
         <property name="title">
          <string>Z Axes Force</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_16">
          <item>
           <widget class="QGraphicsView" name="ZForce1Graph"/>
          </item>
          <item>
           <widget class="QGraphicsView" name="ZForce2Graph"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="ZAxesPositionGroup">
         <property name="title">
          <string>Z Axes Position</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QGraphicsView" name="ZPos1Graph"/>
          </item>
          <item>
           <widget class="QGraphicsView" name="ZPos2Graph"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="XPositionGroup">
         <property name="title">
          <string>X Axis Position</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QLCDNumber" name="XPositionDisp"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="ControlTabBox">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="foo_device">
       <attribute name="title">
        <string>Device Control</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QGroupBox" name="foo_vertical">
          <property name="title">
           <string>Vertical Axes Control</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QGroupBox" name="VerticalAxis1ControlBox">
             <property name="title">
              <string>Axis 1</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_13"/>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="VerticalAxis2ControlBox">
             <property name="title">
              <string>Axis 2</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_14"/>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="HorizontalAxisControlBox">
          <property name="title">
           <string>Horizontal Axis Control</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="foo_macro">
       <attribute name="title">
        <string>Macro Programming</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>862</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRoller_Lamination_Controller">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_macro"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Data"/>
   </widget>
   <addaction name="menuRoller_Lamination_Controller"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open macro</string>
   </property>
  </action>
  <action name="actionSave_macro">
   <property name="text">
    <string>Save macro</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save a copy</string>
   </property>
  </action>
  <action name="actionSave_Data">
   <property name="text">
    <string>Save Data</string>
   </property>
  </action>
  <action name="actionConfigure_Communication">
   <property name="text">
    <string>Configure Communication</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
